diff -ruN libgeotiff-1.7.1/configure libgeotiff-1.7.1-patched/configure
--- libgeotiff-1.7.1/configure	2022-03-08 10:51:12.000000000 -0500
+++ libgeotiff-1.7.1-patched/configure	2023-02-07 15:24:24.000000000 -0500
@@ -13083,7 +13083,7 @@
   ac_status=$?
   $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
   test $ac_status = 0; }; then
-  pkg_cv_PROJ_LIBS=`$PKG_CONFIG --libs "proj" 2>/dev/null`
+  pkg_cv_PROJ_LIBS=`$PKG_CONFIG --static --libs "proj" 2>/dev/null`
 		      test "x$?" != "x0" && pkg_failed=yes
 else
   pkg_failed=yes
