diff -ruN zstd-1.5.2/lib/Makefile zstd-1.5.2-patched/lib/Makefile
--- zstd-1.5.2/lib/Makefile	2022-01-20 16:17:18.000000000 -0500
+++ zstd-1.5.2-patched/lib/Makefile	2023-02-06 15:13:21.000000000 -0500
@@ -308,8 +308,8 @@
 	        $< >$@
 
 .PHONY: install
-install: install-pc install-static install-shared install-includes
-	@echo zstd static and shared library installed
+install: install-pc install-static install-includes
+	@echo zstd static library installed
 
 .PHONY: install-pc
 install-pc: libzstd.pc
